scaffolder-templates/build/.gitlab-ci.yml

```diff
@@ -0,0 +1,25 @@
+stages:
+  - update
+
+update_pipelinerun:
+  tags:
+    - shared-podman # Change to whatever tag your Gitlab runner is set to 
+  stage: update
+  image: ubuntu:latest
+  script: |
+    apt-get update && apt-get install -y git
+    apt-get install apt-transport-https ca-certificates -y 
+    update-ca-certificates
+    git config --global http.sslVerify "false"
+    COMMIT_AUTHOR_EMAIL=$(git log -1 --pretty=format:'%ae')
+    COMMIT_AUTHOR_NAME=$(git log -1 --pretty=format:'%an')
+    COMMIT_ID=$(git rev-parse --short HEAD)
+    git config --global user.email "$COMMIT_AUTHOR_EMAIL"
+    git config --global user.name "$COMMIT_AUTHOR_NAME"
+    git remote set-url origin https://token:$PROJECT_ACCESS_TOKEN@$CI_SERVER_HOST/$CI_PROJECT_PATH.git
```

Comment:
`CI_SERVER_HOST` and  `CI_PROJECT_PATH` env variables are automatically made available by the GL runner when the pipeline is executed, right?

Nice creating `PROJECT_ACCESS_TOKEN` automatically :)

Reply:
Yes, CI_SERVER_HOST, CI_PROJECT_PATH are avaliable automatically
PROJECT_ACCESS_TOKEN is both the name of the custom GL variable and the Token  (and thanks)
and "token" can literally be any non-empty string.